web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
```

Comment:
The `display: flex` and `flex-flow: column nowrap` properties here can both be dropped.

Reply:
Nice catch. Done.